array-manipulation-2/src/take-right.ts

```diff
@@ -0,0 +1,11 @@
+/* exported takeRight */
+function takeRight(array: any[], count: number): unknown[] {
```

Comment:
It's best not to use the `any` type. If you don't care what type the value is, instead of `any`, you should use `unknown`.